Moves styling to style.css and implements Bootstrap

ewmcelvaney opened the pull request:
The way that the text has been displayed has been altered with Bootstrap classes and some styling elements have been removed to reflect this

ewmcelvaney committed:
Moves styling to style.css and implements Bootstrap

The way that the text has been displayed has been altered with Bootstrap classes and some styling elements have been removed to reflect this

elstamey commented:
Thank you so much for your contribution, @ewmcelvaney !!! :tada: 

@cmgmyr Do you know why there's a workflow error?

cmgmyr commented on 404.html:
```diff
@@ -1,25 +1,14 @@
-    letter-spacing: -1px;
-  }
-</style>
+<!--permalink: /404.html
```
I think Jekyll will still need this for the permalink location

cmgmyr commented on 404.html:
```diff
@@ -1,25 +1,14 @@
-  }
-</style>
+<!--permalink: /404.html
+layout: default
```
I think we're at a crossroads with this one. If we remove the `default` layout, then this page will no longer show the header, footer, links, etc. So I think we can either 

1. Keep this page with no layout and keep bootstrap (like you have), but add a link to go back to the home page
2. Find a Bootstrap (or similar) styled theme, use that, then re-style the 404 page.

I'm ok with option 1 since it would be less work and we can get the PR merged for you 🙂

cmgmyr reviewed (changes requested):
@ewmcelvaney Thanks so much for your PR to the project, and great meeting you last night!!! I added a few comments and options for you in order to get the PR merged. If you have any questions, feel free to respond here or ping me in Slack

@elstamey it looks like because this PR was coming from a fork, GH actions is trying to run it there and not in our repo so the token is invalidated. Apparently, it's a known issue but no solution or workaround at this point. I'll probably have to remove it 😢(it worked for you because you had write access to the repo and made a local branch, not a fork)

ewmcelvaney committed:
Tests Git

Tests if I am working in the right copy of this file

ewmcelvaney committed:
Undoes test commit

Removes the added letter from the test commit

ewmcelvaney committed:
Uncommments links and layout

The permalink and the layout are back to being regular lines of code

ewmcelvaney committed:
Removes doctype declaration

I discovered we were writing the site mostly in Ruby with Jekyll, so I this commit removes the doctype declaration to see if the link will play with the site better

ewmcelvaney committed:
Adds the doctype declaration back

Removing the doctype declaration does nothing, so this commit puts it back in place

ewmcelvaney committed:
Introduces column structure to the page

This commit adds in Bootstrap's column structure since I forgot to add it the first time

ewmcelvaney committed:
Shifts the text over a column

Shifts the text of 404.html over one column on Bootstraps' 12-column grid layout

ewmcelvaney committed:
Adds link button

Adds a link button to go back to the home page

ewmcelvaney committed:
Alters button text size

Changes the h1 tags for the link button to h5 tags

ewmcelvaney committed:
Alters one of the 404 messages

The last line of text is changed from repeating that the page could not be found to repeating it while suggesting what might have went wrong and how they can get off the 404 page

ewmcelvaney committed:
Links button to home page

When the button is clicked, it will now link to the home page of the website

ewmcelvaney committed:
Centers the entire page

The entire page is now centered by moving all contents to column 12 on the Bootstrap grid

ewmcelvaney committed:
Adds more text for Jekyll

Additional text at the top was added for Jekyll

ewmcelvaney committed:
Link button color changed

The link button's color is changed from teal to yellow to better reflec

ewmcelvaney committed:
Changes container-fluid to container

Container changes have been reverted

ewmcelvaney committed:
Centers home page button

Centers the button that links to the home page

ewmcelvaney commented:
I went back and changed the code to reintroduce structure to the 404 page and I added a link to go back to the home page just in case

cmgmyr commented:
awesome, thanks!
